HW_9/task_1.py

```diff
@@ -0,0 +1,44 @@
+class Company:
+
+    def __init__(self, name: str, number_of_employees: int, country: str, year_of_creation: int):
+        self.name = name
+        self.number_of_employees = number_of_employees
+        self.country = country
+        self.year_of_creation = year_of_creation
```

Comment:
please watch this lection again. I told you that it is not good to have public variables for class. They should be private or protected. 

When you will make it private, you should create @properties for them (getter and setter) with different limitations